src/matcha_ml/cli/cli.py

```diff
@@ -110,10 +116,17 @@ def get(
 
 @app.command()
 @track(event_name=AnalyticsEvent.DESTROY)
-def destroy() -> None:
-    """Destroy the provisioned cloud resources. It will destroy the resource group even if resources are provisioned inside the group."""
+def destroy(
+    full: Optional[str] = typer.Argument(None),
+) -> None:
+    """Destroy the provisioned cloud resources."""
```

Comment:
The previous docstring is correct. We are force destroying some of the partial resources by default.

Reply:
It will not destroy the resource group unless `matcha destroy` is run with the `full` argument.